go.mod

```diff
@@ -1,21 +1,18 @@
-module github.com/openshift/oadp-non-admin
+module github.com/migtools/oadp-non-admin
 
 go 1.21
 
 require (
+	github.com/go-logr/logr v1.4.1
 	github.com/onsi/ginkgo/v2 v2.14.0
 	github.com/onsi/gomega v1.30.0
```

Comment:
I've added anything manaualy added post generate to second group, is that wrong?

Reply:
without comments yes, we can not know why they are separate